en/docs/get-started/try-out-flow.md

```diff
@@ -77,53 +77,43 @@ Given below is a summary of configurations to follow when deploying the APIs in
    
 ### Authorizing a consent
 
-The Data Recipient application redirects the bank customer to authenticate and approve/deny application-provided consents.
+The Accredited Data Recipient application redirects the bank customer to authenticate and approve/deny application-provided consents.
 
 1. Generate the request object by signing the following JSON payload using supported algorithms.
 
     ``` tab="Format"
     {
-      "kid": "<CERTIFICATE_FINGERPRINT>",
-      "alg": "<SUPPORTED_ALGORITHM>",
+      "kid": "2MI9XSKi6ddxCbWg2rhDNtUlxJc",
```

Comment:
since this is the format, let's use
""kid": "<CERTIFICATE_FINGERPRINT>",
      "alg": "<SUPPORTED_ALGORITHM>","

Reply:
Fix review comments